ios/Sources/SettingFeature/SettingFeature.swift

```diff
@@ -0,0 +1,31 @@
+import ComposableArchitecture
+
+public struct SettingState: Equatable {
+    var darkModeIsOn: Bool
+    var languageIsOn: Bool
+
+    public init(darkModeIsOn: Bool, languageIsOn: Bool) {
```

Comment:
Because we get these values from local database, we don't need to init values.
```suggestion
    public init(darkModeIsOn: Bool = false, languageIsOn: Bool = false) {
```